packages/components/src/common/typography.module.css

```diff
@@ -55,6 +55,10 @@
   @apply text-brand-700;
 }
 
+.colorCurrent {
+  color: currentColor;
```

Comment:
`text-color` [should be a thing ](https://tailwindcss.com/docs/text-color#class-reference) but tailwind didnt seem to recognize it? 🤔

Reply:
Weird. Maybe our color customization clobbers text-transparent and text-color?

Reply:
yeah, it only allows the EDS color tokens right now and 'transparent'. We can add currentColor though:
```
// in tailwind.config.js
module.exports = {
  theme: {
    colors: {
      current: 'currentColor',
      ...
```